src/operations/explore_operation.cpp

```diff
@@ -9,12 +9,20 @@
 #include "util.h"
 
 ExploreOperation::ExploreOperation(const std::vector<geometry_msgs::Point>& path)
-    : MoveOperation(OperationIdentifier::EXPLORE, path, 500, 3, 3),
+    : MoveOperation(OperationIdentifier::EXPLORE, path, 500, 0.5, 1),
       obstacle_avoidance_path_publisher(node_handle.advertise<ascend_msgs::Path>("/obstacle_avoidance/path", 10)),
       obstacle_avoidance_path_subscriber(
           node_handle.subscribe("/obstacle_avoidance/corrected_path", 10, &ExploreOperation::pathCallback, this)) {}
 
 void ExploreOperation::initialize() {
+    if (path.size() == 0) {
+        point_of_interest = getCurrentPose().pose.position;
```

Comment:
Why not, but have you discuss that with AI? I was thinking perhaps that it could be 1m in front of the drone as well. But I don't think there is any perfect solution of course. Probably yours is the best :)

Reply:
It will now face a point 1 m ahead (in the case of empty path) to avoid the possible oscillation around x, y = 0, +-0 in atan2(x, y)